app/controllers/admin/rents_controller.rb

```diff
@@ -0,0 +1,81 @@
+class Admin::RentsController < ApplicationController
+  before_action :login_required
+  before_action :authenticate
+  before_action :set_rent, only: [:show, :update, :edit]
+  before_action :set_rents, only: [:edit, :new]
+  before_action :set_councils, only: [:new, :edit]
+
+  def main
+    @rents = Rent.ascending.from_date(Time.zone.now.beginning_of_day)
+    @rent_grid = initialize_grid(@rents)
+    @faqs = Faq.where(answer: '').where(category: 'Bil')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.